go-controller/pkg/ovn/logical_switch_manager/logical_switch_manager.go

```diff
@@ -306,9 +306,9 @@ func NewJoinIPAMAllocator(cidr *net.IPNet) (ipam.Interface, error) {
 
 // Initializes a new join switch logical switch manager.
 // This IPmanager guaranteed to always have both IPv4 and IPv6 regardless of dual-stack
-func initJoinLogicalSwitchIPManager(existingNodeNames []string) (*joinSwitchIPManager, error) {
-	j := joinSwitchIPManager{
-		lsm: &logicalSwitchManager{
+func InitJoinLogicalSwitchIPManager(existingNodeNames []string) (*JoinSwitchIPManager, error) {
```

Comment:
@alexanderConstantinescu can we call this `NewJoinLogicalSwitchIPManager` instead?

Reply:
Done!